src/__init__.py

```diff
@@ -107,10 +107,10 @@ def bql_quote_name(name):
     'bayesdb_read_csv',
     'bayesdb_read_csv_file',
     'bayesdb_register_metamodel',
+    'bayesdb_upgrade_schema',
     'bql_quote_name',
     'IBayesDBMetamodel',
     'IBayesDBTracer',
-    '__version__',
```

Comment:
What was wrong with this? I'd like to be able to rely on bayeslite.__version__. It's pretty standard.